src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -12,20 +12,48 @@
                 <div class="sidebar-container d-flex h-100 justify-content-between flex-column" [ngClass]="{ collapsed: isNavbarCollapsed }">
                     <!-- NavItems -->
                     <div>
-                        <div class="d-flex p-3 align-items-center text-decoration-none" [title]="course?.title">
-                            @if (course && course.courseIcon) {
-                                <div>
-                                    <jhi-secured-image [src]="course.courseIcon" />
-                                </div>
-                            } @else {
-                                <div class="course-circle d-flex align-items-center justify-content-center">
-                                    <span class="fs-4">{{ course?.title | slice: 0 : 1 }}</span>
-                                </div>
-                            }
+                        <div #container id="container" class="d-flex p-3 align-items-center text-decoration-none" [title]="course?.title">
+                            <div ngbDropdown class="d-inline-block">
+                                @if (course && course.courseIcon) {
+                                    <div class="d-flex align-items-center justify-content-center">
+                                        <jhi-secured-image id="courseIcon" [src]="course.courseIcon" (click)="onIconClick()" />
+                                    </div>
+                                } @else {
+                                    <div id="courseIconCustom" class="course-circle d-flex align-items-center justify-content-center" (click)="onIconClick()">
```

Comment:
please rename the function  `onIconClick()` to something like `toggleCourseDropdown()`

Reply:
done